data/sql/updates/pending_db_world/rev_1653580726731163907.sql

```diff
@@ -0,0 +1,3 @@
+
+UPDATE `creature_template` SET `ScriptName`='npc_chained_spirit', `speed_run`=0.4 WHERE `entry`=15117;
```

Comment:
source for speed_run change?

Reply:
@Nefertumm https://youtu.be/lwQWch4YUgE?t=255